packages/web-components/src/components/card/defs.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export enum PICTOGRAM_POSITION {
+  /**
+   * Top pictogram
+   */
+  TOP = 'top',
+  /**
+   * Bottom pictogram
+   */
+  BOTTOM = 'bottom',
+}
+
+export default PICTOGRAM_POSITION;
```

Comment:
To align to `ICON_PLACEMENT` for `<dds-card-footer>`:

```suggestion
export default PICTOGRAM_PLACEMENT;
```